pages/user/signin.vue

```diff
@@ -1,9 +1,5 @@
 <template>
   <div class="ma-10">
-    <!-- if the user just registered a msg should be displayed in a chip -->
-    <p v-if="$route.query.registered === 'true'">
```

Comment:
This needs to be replaced with an alert. When redirected to signin after user creation the alert should tell you the user was created. Similarly after editing your own user, the alert should display success.